resources/META-INF/scala-plugin-common.xml

```diff
@@ -920,6 +920,10 @@
                          displayName="Concealed &quot;apply&quot; call" groupPath="Scala" groupName="Syntactic Clarification"
                          shortName="ConcealedApplyCall" level="WARNING"
                          enabledByDefault="false" language="Scala"/>
+        <localInspection implementationClass="org.jetbrains.plugins.scala.codeInspection.syntacticClarification.ConvertNullInitializerToUnderscore"
+                         displayName="Null initializer can be replaced by _" groupPath="Scala" groupName="Syntactic Clarification"
+                         shortName="ConvertNullInitializerToUnderscore" level="WARNING"
+                         enabledByDefault="true" language="Scala"/>
```

Comment:
The main issue with the new inspections disabled by default that most users will miss them due to lack of normal changlelog. But the enabled inspection can always be disabled manually.